src/one_dragon_qt/widgets/download_card/launcher_download_card.py

```
@@ -90,53 +74,27 @@ def __init__(self, ctx: OneDragonEnvContext):
-
-    @property
-    def _zip_suffix(self) -> str:
-        return RUNTIME_LAUNCHER_ZIP_SUFFIX if self._launcher_type == 'runtime' else LAUNCHER_ZIP_SUFFIX
-
-    @property
-    def _is_runtime(self) -> bool:
-        return self._launcher_type == 'runtime'
-
     def _on_type_changed(self, _index: int) -> None:
         self._launcher_type = self.type_combo.currentData()
         # 断开旧检查器信号，避免竞态覆盖
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)